src/components/Accordion/Accordion.tsx

```diff
@@ -34,19 +38,21 @@ export default class Accordion extends React.Component<
     }
 
     render() {
-        const { buttonOptions, modifiers = [], blockName = "" } = this.props;
-        buttonOptions.type = "button";
-        buttonOptions.blockName = "accordion";
-        buttonOptions.modifiers = ["large"];
-        buttonOptions.iconPosition = ButtonIconPositions.JustifyRight;
-        buttonOptions.iconName = this.state.isOpen ? "minus" : "plus";
-        buttonOptions.iconModifiers = ["medium"];
-        buttonOptions.buttonType = ButtonTypes.Outline;
+        const { modifiers = [], blockName = "" } = this.props;
+
+        let ButtonProps: { [key: string]: any } = {};
+        ButtonProps.id = this.props.labelId;
+        ButtonProps.type = "button";
+        ButtonProps.blockName = "accordion";
+        ButtonProps.iconPosition = ButtonIconPositions.JustifyRight;
+        ButtonProps.iconName = this.state.isOpen ? "minus" : "plus";
+        ButtonProps.iconModifiers = ["medium"];
+        ButtonProps.buttonType = ButtonTypes.Secondary;
 
         return (
             <div className={bem("accordion", modifiers, blockName)}>
-                <Button callback={this.toggleContentShow} {...buttonOptions}>
-                    {buttonOptions.content}
+                <Button onClick={this.toggleContentShow} {...ButtonProps}>
```

Comment:
I know this was here before, but I think we should be explicit about props instead of passing them all in like this - IMO it's more clear.

Reply:
I'm not sure I understand—what do you mean by being explicit, and what does unintended circumstances does it avoid?

Reply:
Like going 
`<button id={blah} className={blah} onClick={blah} >` 

instead of assigning them above and passing it as `{...ButtonProps}` 

Personally I find it easier to read, but also it makes sure that you're not passing in anything that you don't intend to.  I think we should only use the spread syntax to make it clear that we aren't completely sure what we're passing in (like attributes).